### PR TITLE
Updates to docs for using Globus on Aurora 

### PR DESCRIPTION
**Pull request by rickybalin**
Added some more details to the existing page on how to use Globus on the Aurora logins to facilitate the process for users.

### CODE REVIEW

File: docs/aurora/data-management/moving_data_to_aurora/globus.md [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
+Perform the following steps to transfer data to/from the Aurora login nodes.
 
+1. On a fresh connection to the login nodes, ensuring no proxies are being set (which may require commenting the proxy settings in the `~/.bashrc` or `~/.bash_profile` files), execute
 ```bash
 /soft/tools/proxychains/bin/proxychains4 -f /soft/tools/proxychains/etc/proxychains.conf /soft/tools/globusconnect/globusconnect -setup --no-gui
 ```
-and follow the instruction to setup personal endpoint. 
 
+2. Paste the link provided by the above command in a browser and follow the instructions to setup a personal endpoint
+	* When requested, input your ALCF username and one-time password from your CRYPTOCard/MobilePASS+ token
```
Comment (felker) [inline docs/aurora/data-management/moving_data_to_aurora/globus.md:12]: spacing issue?

File: docs/aurora/data-management/moving_data_to_aurora/globus.md [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
+
+3. On the same terminal, execute 
 ```bash
 /soft/tools/proxychains/bin/proxychains4 -f /soft/tools/proxychains/etc/proxychains.conf /soft/tools/globusconnect/globusconnect -start &
 ```
-You can also add -restrict-paths /lus/flare/projects/YOURPROJECT  to access folders outside of your home
+    * By default, the command only gives access to your home directory
+    * You can add `-restrict-paths /lus/flare/projects/YOURPROJECT` to access your project directory
+
+4. Open the [GLobus web app](https://app.globus.org/file-manager?destination_id=05d2c76a-e867-4f67-aa57-76edeb0beda0) and search for the endpoint name defined above. You will now see your home directory (and project directory, if requested) on Aurora and can initiate transfers with other endpounts (e.g., the Eagle file system on Polaris at `alcf#dtn_eagle`)
```
Comment (felker) [inline docs/aurora/data-management/moving_data_to_aurora/globus.md:24]:
```suggestion
4. Open the [Globus web app](https://app.globus.org/file-manager?destination_id=05d2c76a-e867-4f67-aa57-76edeb0beda0) and search for the endpoint name defined above. You will now see your home directory (and project directory, if requested) on Aurora and can initiate transfers with other endpoints (e.g., the Eagle file system on Polaris at `alcf#dtn_eagle`)
```

File: docs/aurora/data-management/moving_data_to_aurora/globus.md [hunk 3]
```diff
@@ -1,13 +1,25 @@
...
 ```bash
 /soft/tools/proxychains/bin/proxychains4 -f /soft/tools/proxychains/etc/proxychains.conf /soft/tools/globusconnect/globusconnect -setup --no-gui
 ```
-and follow the instruction to setup personal endpoint. 
 
+2. Paste the link provided by the above command in a browser and follow the instructions to setup a personal endpoint
+	* When requested, input your ALCF username and one-time password from your CRYPTOCard/MobilePASS+ token
+    * Select the Allow button
+    * Enter the authentication code generated back into the terminal
+    * Enter a name for the endpoint (e.g., aurora\_login\_uan11)
```
Comment (felker) [inline docs/aurora/data-management/moving_data_to_aurora/globus.md:15]:
```suggestion
    * Enter a name for the endpoint (e.g., `aurora_login_uan11`)
```

**Review by felker — CHANGES_REQUESTED** [review]

File: docs/aurora/data-management/moving_data_to_aurora/globus.md (hunk 1 above)
Comment (rickybalin) [reply docs/aurora/data-management/moving_data_to_aurora/globus.md:12]: This was to create a sub-bullet up item 2. It visualized well for me in the browser when I tested it.

**Review by felker — APPROVED** [review]

**Pull request merged**